src/gt4py/definitions.py

```diff
@@ -513,8 +510,8 @@ def shape(self):
 
 
 class Extent(FrameTuple):
-    """Stencil support: region defined by the smallest and the largest offsets in
-     a stencil pattern computation.
+    """
```

Comment:
Comestic detail: I think in this codebase docstrings start always (or most of the times) in the triple-quoted line